### PR TITLE
CodeMouse92's Ubuntu Cheatsheet Goodie

### PR DESCRIPTION
**Pull request by CodeMouse92**
# Goodie Pull Request Template

**What does your Instant Answer do?**
It displays the common keyboard shortcuts for Linux Ubuntu Unity.

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Even though many of these shortcuts can be accessed by holding down the Super key on Ubuntu, that list is A) easily forgotten about for new users, B) somewhat incomplete, and C) inaccessible for someone who is providing tech help over the phone to an Ubuntu user, but is away from Ubuntu at the moment.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Ubuntu itself - hold down the Super key. The rest are found on System Settings -> Keyboards -> Shortcuts.

**Why did you choose this data source?**
It was built in.

**Are there any other alternative (better) data sources?**
Not really. Even the online docs don't match up 100%.

**What are some example queries that trigger this Instant Answer?**
ubuntu cheatsheet, ubuntu cheat sheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Linux users.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No.

**Which existing Instant Answers will this one supercede/overlap with?**
None.

**Are you having any problems? Do you need our help with anything?**
Not that I can tell.

**Where did you hear about DuckDuckHack? (For first time contributors)**
While browsing the DuckDuckGo website. I LOVE this search engine!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

cheatsheet

## Checklist
Please place an 'X' where appropriate. (I followed https://duck.co/duckduckhack/goodie_cheat_sheets)

```
- [X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [X] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [X] Tested cross-browser compatibility

I only have Ubuntu, so I checked Firefox and Chromium.

    Please let us know which browsers/devices you've tested on:
    - Ubuntu 15.04
        - [X] Google Chrome
        - [X] Firefox
        - [] Opera
```
-----
IA Page: https://duck.co/ia/view/ubuntu_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/ubuntu.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        "Screenshots"
+    ],
+    "sections": {
+        "Dash": [
+            {
+                "key": "[Super]",
+                "val": "Opens Dash"
+            },
+            {
+                "key": "[Super] + [A]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/ubuntu.json]: Please remove the `+`s from all these. We recently update the syntax and our docs to reflect this, but I think some bad example may still be lingering?

i.e.`[Super] [A]`

File: share/goodie/cheat_sheets/ubuntu.json [hunk 2]
```diff
@@ -0,0 +1,161 @@
+{
+    "id": "ubuntu_cheat_sheet",
+    "name": "Ubuntu Unity Shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/ubuntu.json]: Let's remove "Shortcuts" from the `name`

**Comment by moollaza** [comment]
@CodeMouse92 thanks a lot for contributing, we really appreciate it! I've left a little bit of feedback. Once those are addressed I think this will be good to go!

**Comment by CodeMouse92** [comment]
I updated, though I have no idea what to do about the failed integration.

**Comment by mintsoft** [comment]
@CodeMouse92 The fix the those tests is in master now, if you rebase it should go green

**Comment by CodeMouse92** [comment]
Thank you very much! Updated, and the tree is green. Good day to you. :)

**Comment by moollaza** [comment]
@CodeMouse92 This LGTM :+1:

**Comment by CodeMouse92** [comment]
I'll certainly be around! I'm hooked.

**Comment by moollaza** [comment]
@CodeMouse92 congrats! It's live: https://duckduckgo.com/?q=ubuntu+unity+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**